### ISSUE TITLE
Minus Sign font-awesome

### ISSUE THREAD

**Issue by ThomasPerraudin** [kartik-v/bootstrap-star-rating#126]
Minus Sign is not displayed when using Font Awesome theme. (bootstrap glyphs not loaded in favor of font awesome)

**Comment by ThomasPerraudin** [comment]
Temporarily fixed with CSS:
```
.rating-container .glyphicon-minus-sign {
  box-sizing: border-box;
  position: relative;
  top: 1px;
  display: inline-block;
  font-family: 'Glyphicons Halflings';
  font-style: normal;
  font-weight: 400;
  line-height: 1;
  -webkit-font-smoothing: antialiased;
}
.rating-container .glyphicon-minus-sign:before {
  content: "\24ea"; /* a zero into a circle */
}
```

**Comment by kartik-v** [comment]
Will be solved via #130 --- update to release v4.0.2.
